### PR TITLE
FIX: Bubble Message Layout in Conversation when Appending Partial via Turbo Stream

### LINKED ISSUE #38: FIX: Conversation Bubble Message
- Fixes the conversation bubble message layout due to the nature of websockets

### PR DESCRIPTION
**Pull request by reyolan**
closes #38 

- Fixed layout already
- Needed to do unconventional workarounds (especially putting data attribute in head element) to make it work hahahhahahaha!
- Yung sa seen, hindi ko pa muna priority, will revisit it after we're done with MVP.

### CODE REVIEW

File: app/views/conversation_messages/_message.html.erb [hunk 1]
```diff
@@ -1,9 +1,9 @@
-<div class="flex flex-col <%= message.user == user ? "col-start-2 col-end-5" : "col-start-1 col-end-4" %>" id="<%= dom_id(message)%>">
-  <div class="flex-1 p-4 rounded-t-full <%= message.user == user ? "bg-green-600 text-white rounded-bl-full" : "bg-gray-200 text-gray-900 rounded-t-full rounded-br-full" %>">
+<div class="flex flex-col <%= message.user == user ? "col-start-2 col-end-5" : "col-start-1 col-end-4" %>" id="<%= dom_id(message)%>" data-hidden="<%= owned_by %>">
```
Comment (superspike7) [inline app/views/conversation_messages/_message.html.erb:1]: para saan itong `data-hidden`?

**Review by superspike7 — APPROVED** [review]
nice one for making this one working!

di na ako mag redesign hahaha

File: app/views/conversation_messages/_message.html.erb (hunk 1 above)
Comment (reyolan) [reply app/views/conversation_messages/_message.html.erb:1]: ayan yung way para ma display:none ko sa screen ng sender yung bubble message, ang ginawa ko kasing workaround is kada send ng message, nagreredirect. Then, with that data-hidden, hidden sa sender yung sinend na message via websocket. 

So magrerender lang yung partial template data na sinend via websocket sa screen ng receiver. Dahil naka display:none yung sa sender side, need ko iredirect kada send ng message para makita niya yung sinend niyang message.

Comment (superspike7) [reply app/views/conversation_messages/_message.html.erb:1]: oooooh nice good job pre! galing galing

**Pull request merged**